DataImport.Web/Infrastructure/MvcTransactionFilter.cs

```diff
@@ -65,16 +64,6 @@ private static void HandleOdsApiServerException(ActionExecutedContext filterCont
                             { "Id", odsApiServerException.ApiServerId.Value }
                     });
             }
-            else
-            {
-                result = new RedirectToRouteResult(
-                    new RouteValueDictionary
-                    {
-                        { "controller", "Configuration" },
-                        { "action", "Index" },
-                        { nameof(EditConfiguration.Query.OdsApiServerException), true }
-                    });
-            }
```

Comment:
@lchavarriaga
Please consider to verify the flow - if no api server connections defined, then the page needs to be redirected to API-connections page.

Reply:
Fixed

Reply:
Fixed